Library/Formula/swiftgen.rb

```diff
@@ -0,0 +1,19 @@
+class Swiftgen < Formula
+  desc "A collection of Swift tools to generate Swift code (enums for your assets, storyboards, Localizable.strings, …)"
```

Comment:
This is going to be *way* too long. The limit is 80 characters, including the name. Will be highlighted by `brew audit --strict`